data/ActType.csv

```diff
@@ -1,6 +1,9 @@
 action_ID,action_name,created,created_by,last_modified,last_modified_by
 AT01,to read,2017-07-12,OS,2017-07-12,OS
-AT02,to copy,2017-07-12,OS,2017-07-12,OS
+AT02,to copy (shouchao),2017-07-12,OS,2017-07-28,LH
```

Comment:
I actually like the idea of having a typography of shuochao practices in here. Still think we shouldn't cope out so easily by using pinyin. if there are different ways of transcribing you want to clearly distinguish we need to find terms for them. Probably better to open an issue for the discussion though.